int32 ACTION_SELECT=0
int32 ACTION_CLOSE=1
int32 action
uint32 current_index
string[] menus
string title
std_msgs/ColorRGBA bg_color
std_msgs/ColorRGBA fg_color
